docs/en/setup/backend/ui-grafana.md

```diff
@@ -101,6 +101,14 @@ to prevent it value be escaped.
 4. Test query and save the panel.
 <img src="https://skywalking.apache.org/screenshots/9.6.0/logql/grafana-logs-panel.jpg"/>
 
+## Configure General Relation Dashboards
+The following steps are the example of config a `General Relation` dashboard:
+1. Create a dashboard named `General Relation`.
+2. Configure variables for the dashboard:
+   <img src="https://skywalking.apache.org/screenshots/10.1.0/promql/grafana-relation-variables1.jpg"/>
+   After configure, you can select the service/instance/endpoint and the dest service/instance/endpoint on the top of the dashboard:
+   <img src="https://skywalking.apache.org/screenshots/10.1.0/promql/grafana-relation-variables2.jpg"/>
```

Comment:
I think you should add an example dashboard config screenshot with the config step. You can refer the other section above.

Reply:
here is only variables

Reply:
Got it, But I think the rest is almost the same as the general dashboard, how should I phrase it (just write 'same as general' or add a new part)?

Reply:
The PrpmQL labels are different, we need to tell users how to config a relation-metrics PromQL